packages/zosjobs/src/DownloadJobs.ts

```diff
@@ -104,11 +101,7 @@ export class DownloadJobs {
         ImperativeExpect.keysToBeDefined(parms, ["jobFile"], "You must specify a job file on your 'parms' parameter" +
             " object to the downloadSpoolContentCommon API.");
 
-        if (parms.outDir == null) {
-            parms.outDir = DownloadJobs.DEFAULT_JOBS_OUTPUT_DIR;
-        }
-
-        const file = DownloadJobs.getSpoolDownloadFile(parms.jobFile, parms.omitJobidDirectory, parms.outDir);
+        const file = DownloadJobs.getSpoolDownloadFilePath(parms);
```

Comment:
Smart way to deprecate the `getSpoolDownloadFile` method. 👍 
I was initially concerned about the outDir behavior possibly changing, but it makes sense to more that login into the new version of the function